R/utilities.R

```diff
@@ -30,7 +30,7 @@
     content <- .strip(content(response, "text"))
     if (nchar(content) == 0)
         return(character(0))
-    do.call(parser, list(content, ...))
+        do.call(parser, list(content, ...))
```

Comment:
Please remove the indentation here. This code would get run when `nchar(content) == 0` is `FALSE` so the original indentation was appropriate.

Reply:
That was an accident, thanks for pointing it out!